components/EmailValidator.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="d-inline-block w-100">
+    <validating-form
+      :class="{ 'justify-content-around': center, 'd-flex': center }"
+    >
+      <b-form-group
+        :label="label"
+        :label-for="'email-' + id"
+      >
+        <validating-form-input
+          :value="email"
+          type="email"
+          name="email"
+          :size="size"
+          class="email"
+          :validation="$v.email"
+          validation-enabled
+          :validation-messages="{
+            email: 'Please enter a valid email address.'
+          }"
+          :center="center"
+          autocomplete="username email"
+          @input="input"
+          @focus="focus"
+          @blur="blur"
+        />
+      </b-form-group>
+    </validating-form>
+    <div v-if="suggestedDomains && suggestedDomains.length" class="text-info small">
+      Did you mean <b>{{ suggestedDomains[0] }}</b>?
+    </div>
+  </div>
+</template>
+<script>
+import { required, email } from 'vuelidate/lib/validators'
+import { validationMixin } from 'vuelidate'
+import ValidatingForm from '../components/ValidatingForm'
+import ValidatingFormInput from '../components/ValidatingFormInput'
+import validationHelpers from '@/mixins/validationHelpers'
+
+const a = require('axios')
```

Comment:
What is the reason for the speific require of axios here?

Reply:
None - will use this.$axios.